src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -0,0 +1,21 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+
+<Project>
+ 
+  <UsingTask TaskName="ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)"/>
+  
+  <Target Name="RunPackageValidation" AfterTargets="Pack">
+    <PropertyGroup>
+      <PackagePath Condition="'$(PackagePath)' == ''">$(PackageOutputPath)$(PackageID).$(Version).nupkg</PackagePath>
```

Comment:
Sad that there's no property offered by NuGet that points to the output path of the package: https://github.com/NuGet/Home/issues/6671.

Anyway, wouldn't you need to use `$(PackageOutputAbsolutePath)$(PackageId).$(PackageVersion).nupkg` instead? I don't think `Version` and `PackageVersion` are always in sync. Also it's probably better to use the absolute path here.

Reply:
In addition to that, I would call the property `PackageTargetPath` (the property suggestion used in the opened issue) to avoid any unintentional name overlaps.